### ISSUE TITLE
GroupedOpenApi for different URLs

### ISSUE THREAD

**Issue by Marx2** [springdoc/springdoc-openapi#309]
I want to expose multiple APIs through one endpoint.
Examples using GroupedOpenApi works only if  everything is packed into single application, and just join documentation from different jars.
I want to achieve something similar to http://springfox.github.io/springfox/docs/current/#aggregating-multiple-swagger-specifications-in-the-same-swagger-ui (location can be full URL)
I was trying to use OpenApiCustomiser but it force me to set paths or group
`Packages to scan or paths to filter can not be both null for the group:ftp`
while I have only external url:

```
	@Bean
	public GroupedOpenApi ftpOpenApi()
	{
		final OpenApiCustomiser customizer = openApi -> {
			final ExternalDocumentation ext = new ExternalDocumentation();
			ext.setUrl("http://localhost:8220/v3/api-docs/");
			openApi.setExternalDocs(ext);
		};
		return GroupedOpenApi.builder().setGroup("ftp").addOpenApiCustomiser(customizer).build();
	}
```

**Comment by bnasslahsen** [comment]
GroupedOpenApi usage, is only for multiple OpenAPI definitions in a single Spring Boot project.

If your purpose is to expose extrernal OpenAPI description only, you can add swagger-config.json (sample attached - containing the urls and group names) to your static ressouces and define the following property:
[swagger-config.txt](https://github.com/springdoc/springdoc-openapi/files/4014304/swagger-config.txt)

```properites
springdoc.swagger-ui.configUrl=/swagger-config.json
```

If you want to combine in the same swagger-ui **both**, REST APIs that are deployed within the same application as the swagger-ui and external REST APIs: This is an enhancement and will be available with the next release: v1.2.22.

To achieve that, you declare your REST APIs that are deployed within the same application as the swagger-ui using GroupedOpenApi.
 And you external REST APIs with the following syntax:
 Note that for serving external urls from the same swagger-ui, you need to have CORS enabled.

```java
SwaggerUiConfigProperties.addGroup("group1", "url1");
SwaggerUiConfigProperties.addGroup("group2", "url2");
```

**Comment by bnasslahsen** [comment]
Enhanced with #584
